source/geometries/Next100FieldCage.cc

```diff
@@ -425,7 +425,7 @@ void Next100FieldCage::BuildCathode()
 
       PlaceHexagons(n_hex, cathode_mesh_diam_, grid_thickn_, cathode_grid_logic, cathode_hex_logic, cathode_int_diam_);
 
-      new G4PVPlacement(0, G4ThreeVector(0., 0., cathode_zpos_), cathode_grid_logic,
+      new G4PVPlacement(0, G4ThreeVector(0., 0., cathode_zpos_ - cathode_thickn_/2. + grid_thickn_/2.), cathode_grid_logic,
```

Comment:
We could use the same variable for the position of the grid, in both cases (dielectric and SS grid), to avoid possible errors, should the position be modified in the future.
Also, `x` and `y` should also be the same, right?

Reply:
I've added the variable declaration for the positon so its common between the two. I believe the x and y should be zero still.